src/main/java/org/jenkinsci/plugins/envinject/EnvInjectConstants.java

```diff
@@ -0,0 +1,16 @@
+package org.jenkinsci.plugins.envinject;
+
+/**
+ * Define the environment variables
+ */
+public final class EnvInjectConstants {
```

Comment:
I think this class should be made restricted, because the API may need some refactoring. <code>@Restricted(NoExternalUse.class)</code>. I can do it after the merge